readme.md

```diff
@@ -4,4 +4,4 @@ Minesweeper is single-player logic-based computer game played on rectangular boa
 
 https://www.figma.com/file/8YyG2KucuKViAzZMDMVunX/Untitled?node-id=0%3A1 - mock ups
 
-https://pastebin.com/vUSXAPGR - 5 functions
\ No newline at end of file
+https://pastebin.com/kYiDr3aD - 5 functions with its description (let me know if more specific description is required)
```

Comment:
It shouldn't be functions, like a coded functions) I need to know ur functionality) like, for example, changing a field size - so for the 4th lab I'll ask u to make a proper radio-button's group to changing size